lib/studies/reply/adaptive_nav.dart

```diff
@@ -514,12 +516,21 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
       duration: const Duration(milliseconds: 350),
       vsync: this,
     );
+
+    _bottomAppBarController = AnimationController(
+      vsync: this,
+      duration: kThemeAnimationDuration,
+    );
+
+    // Initialize with bottom app bar visible
+    _bottomAppBarController.forward();
```

Comment:
Does this need to animate here? What if the animation or animation value was instead initially set to complete / 1?

Reply:
Your right, updated it!